SwiftI18n/Classes/Views/SwiftUI/LocText.swift

```diff
@@ -60,7 +60,7 @@ private extension LocText {
         func text(localizedIn language: String) -> Text {
             switch self {
             case .key(let key):
-                Text(I18nManager.instance.localizationPerformingBlock(key, language))
+                Text(I18nManager.instance.localizedString(forKey: key, language: language))
```

Comment:
Do we need to change this? 🤔

Reply:
If we want it to work, than I would say yes